src/main/resources/l10n/JabRef_sv.properties

```diff
@@ -621,8 +621,8 @@ Level=Nivå
 Left_entry=Vänster_post
 Limit_to_fields=Begränsa_till_fält
 Limit_to_selected_entries=Begränsa_till_valda_poster
-Line_%0\:_Found_corrupted_BibTeX_key.=
-Line_%0\:_Found_corrupted_BibTeX_key_(comma_missing).=
+Line_%0\:_Found_corrupted_BibTeX_key.=Rad_%0\:_Hittade_felaktig_BibTeX-nyckel.
+Line_%0\:_Found_corrupted_BibTeX_key_(comma_missing).=Rad_%0\:_Hittade_felaktig_BibTeX-nyckel_(kommatecken_saknas).
 Line_%0\:_Found_corrupted_BibTeX_key_(contains_whitespaces).=
```

Comment:
Have you overseen this one? Or is the translation of "contains whitespace" hard?